### DISCUSSION TITLE
Purpose of active variable in Reveiver.py

### DISCUSSION THREAD

**Discussion by john-** [lordmorgul/ham2mon#11] [Q&A]
I see commit `d5bc9fc8` introduced the `active` variable to the Receiver class.  However in that commit and since than this variable does not appear to be used.  It is set 3 places in Receiver.py and does not appear to be referenced anywhere else.  Here is one of the 3 places it is [set](https://github.com/lordmorgul/ham2mon/blob/dd5738d06d4c74f5023de04b1d3517547dc0383c/apps/receiver.py#L31).

I temporarily removed it as I am making some changes and I do not see any impact.

Any idea if this is used somehow and I just don't see it?

**Comment by lordmorgul** [comment]
Need to review but I thought that was intended for a UI hint that I did not use.

**Comment by lordmorgul** [comment]
Confirmed it was not needed and just left unused.  This can be deleted where it appears.

**Reply by john-** [reply]
This was resolved with this merged pull request: https://github.com/lordmorgul/ham2mon/pull/14
